src/component/Pagination.js

```diff
@@ -0,0 +1,53 @@
+import { GoArrowRight, GoArrowLeft } from "react-icons/go";
+function Pagination({ nPages, currentPage, setCurrentPage }) {
+  let previousPage;
```

Comment:
Avoid declaring new variables inside component. Maybe instead, wrapping the whole logic into a `useMemo` function. Something like this:
```
const { previousPage, nextPage } = useMemo(() => { ... }, [nPages, currentPage})
```